lib/ansible/plugins/vars/host_group_vars.py

```diff
@@ -122,7 +122,7 @@ def _get_dir_files(self, path):
 
         found = []
         for spath in os.listdir(path):
-            if not spath.startswith(b'.') and not spath.endswith(b'~'):  # skip hidden and backups
+            if not spath.startswith('.') and not spath.endswith('~'):  # skip hidden and backups
```

Comment:
also they can be made into a single tuple:
`if not spath.startswith((u'.',u'~')): `